ui/packages/components/src/FunctionConfiguration/FunctionConfiguration.tsx

```diff
@@ -186,17 +194,35 @@ export function FunctionConfiguration({
             <>
               {concurrencyLimit.limit.value >= 1 && concurrencyLimit.limit.value}
               {concurrencyLimit.limit.isPlanLimit && (
-                <Tooltip>
-                  <TooltipTrigger asChild>
-                    <span>
-                      <Pill className="ml-2">Plan limit</Pill>
+                <Info
+                  side="bottom"
+                  align="end"
+                  text={
+                    <span className="whitespace-pre-line">
+                      Running into limits? Easily upgrade your plan or boost concurrency on your
+                      existing plan.
                     </span>
-                  </TooltipTrigger>
-                  <TooltipContent>
-                    If not configured, the limit is set to the maximum value allowed within your
-                    plan.
-                  </TooltipContent>
-                </Tooltip>
+                  }
+                  widthClassName="max-w-xs"
+                  action={
+                    <Link
+                      href="/billing/plans?ref=concurrency-limit-popover"
```

Comment:
We have a util that is quite helpful for links: https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/utils/urls.ts#L49
We use it so it's easier to replace the references when the paths change